monai/transforms/compose.py

```diff
@@ -170,6 +171,40 @@ def inverse(self, data):
             data = apply_transform(t.inverse, data, self.map_items, self.unpack_items)
         return data
 
+    def get_number_image_type_conversions(self, test_data, key: Optional[Hashable] = None):
```

Comment:
I feel this is just for profiling/usability purposes, not an essential property of Compose, would be great to make it as a utility, which takes a `Compose` instance as input